src/js/components/Tag/Tag.js

```diff
@@ -47,7 +66,7 @@ const Tag = forwardRef(
       console.warn('Tag cannot combine "onClick" and "onRemove".');
     }
 
-    const removeProps = !theme.tag.remove.kind
+    const themeTagRemoveProps = !theme.tag.remove.kind
```

Comment:
I don't think we need to rename this anymore

Reply:
aw yes will revert